connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -494,15 +500,25 @@ private void expectTopicCreation(String topic) {
         }
     }
 
+    private void mockSinkTransform() {
+        FaultyPassthrough<SinkRecord> faultyPassthrough = new FaultyPassthrough<>();
+        @SuppressWarnings("unchecked")
+        Class<? extends Transformation<?>> value = (Class<? extends Transformation<?>>) (Class<?>) FaultyPassthrough.class;
```

Comment:
```suggestion
        Class<? extends Transformation<?>> value = (Class<? extends Transformation<?>>) FaultyPassthrough.class;
```

nit: looks like a redundant type cast?

Reply:
Unfortunately this is a case where the IDE linter is wrong, it won't compile without this cast:

```
error: incompatible types: Class<FaultyPassthrough> cannot be converted to Class<? extends Transformation<?>>
        Class<? extends Transformation<?>> value = (Class<? extends Transformation<?>>) FaultyPassthrough.class;
```

Reply:
Huh, interesting, TIL.